plenum/common/messages/internal_messages.py

```diff
@@ -45,3 +45,21 @@
 
 PreSigVerification = NamedTuple('PreSigVerification',
                                 [('cmsg', Any)])
+
+# by default view_no for StartViewChange is None meaning that we move to the next view
+NeedViewChange = NamedTuple('StartViewChange',
+                            [('view_no', int)])
+NeedViewChange.__new__.__defaults__ = (None,) * len(NeedViewChange._fields)
```

Comment:
Just a small question - why do we need this?

Reply:
To  be able to call NeedViewChange() without parameters and go to the next view. We will see whether we need it during integration phase.